src/cargo/ops/cargo_add/mod.rs

```diff
@@ -966,16 +966,53 @@ fn print_dep_table_msg(shell: &mut Shell, dep: &DependencyUI) -> CargoResult<()>
         } else {
             "".to_owned()
         };
+
         shell.write_stderr(format_args!("{}Features{}:\n", prefix, suffix), &style::NOP)?;
+
+        const MAX_FEATURE_PRINTS: usize = 50;
+
+        let mut activated_printed: usize = 0;
+        let total_activated = activated.len();
         for feat in activated {
+            if activated_printed >= MAX_FEATURE_PRINTS {
+                shell.write_stderr(
+                    format_args!(
+                        "{}... {} more activated features{}\n",
+                        prefix,
+                        total_activated - activated_printed,
+                        suffix
```

Comment:
It makes little sense to print `suffix` here. Maybe we don't need that?

Reply:
Agreed, I failed to realize that suffix is just a version string. I removed it from both.